flask_openapi3/openapi.py

```diff
@@ -79,7 +80,7 @@ def __init__(
             openapi_extensions: Extensions to the OpenAPI Schema.
                                 See https://spec.openapis.org/oas/v3.0.3#specification-extensions.
             validation_error_status: HTTP Status of the response given when a validation error is detected by pydantic.
-                                    Defaults to 422
```

Comment:
[Defaults](https://conjugator.reverso.net/conjugation-english-verb-default.html) in English would be more accurate.
"It defaults to" vs "it default to". 

Not important of course.

Reply:
English is not my native language, thank you for pointing out.

Reply:
No worries, it's not mine either, we're all here to help each other out for a common goal :)